src/viewer/clients/loadReplay.ts

```diff
@@ -15,12 +15,21 @@ const cache: { [key: string]: [ReplayData, ReplayMetadata] } = {}
 export const loadReplay = async (
   replayId: string,
   cached?: boolean,
-  local?: boolean
+  local?: boolean,
+  usecgg: boolean=true
 ): Promise<[ReplayData, ReplayMetadata]> => {
-  const url = local ? "../examples/" : "https://calculated.gg/api/replay/"
+
+  var remoteAddress
+  if (usecgg) {
+    remoteAddress = "https://calculated.gg/api"
+  } else {
+    remoteAddress = location.protocol + '//' + location.host
+  }
+
+  const url = local ? "../examples/" : remoteAddress + "/replay/"
```

Comment:
Priority of `local` and `usecgg` switches isn't clear. I think this is mainly because the `usecgg` switch's computation (wrt `remoteAddress`) is always done, even when it has no effect. Perhaps setting `let url` then modifying it according to `local`/`usecgg` would make it clearer?